Makefile

```diff
@@ -44,6 +44,12 @@ build: ## build all containers
 	$(COMPOSE) build app-dev
 .PHONY: build
 
+reset:  ## Remove postgresql database
+	$(COMPOSE) stop
+	rm -Ir data/* || exit 0
```

Comment:
If you are trying to exit when expecting a `no` answer, this will not work as `rm` exit code will be `0` anyway.